### DISCUSSION TITLE
Add a GUI method of moving the ledger

### DISCUSSION THREAD

**Discussion by luotinen** [Chia-Network/chia-blockchain#8630] [Ideas]
(On Windows) The majority of users have limited space available on C drive. It is filling up rapidly. To maintain the Chia's easy-to-start - easy-to-maintain approach and appeal, I suggest adding a GUI method of moving the ledger onto a different drive. This means that once the ledger fills C and starts to cause problems, the users can just move the ledger onto another drive.

The blockchain_v1_mainnet.sqlite is currently approximately 20 GB as of 29.9.2021.

**Comment by BrandtH22** [comment]
@luotinen , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
